psycop/common/model_training_v2/config/baseline_test_config.cfg

```diff
@@ -0,0 +1,44 @@
+[project_info]
+experiment_path = /.
+
+[logger]
+@baseline_loggers = "terminal_logger"
+
+[trainer]
+@baseline_trainers = "split_trainer"
+training_outcome_col_name = "outcome"
+validation_outcome_col_name = "outcome"
+
+[trainer.training_data]
+@baseline_data = "minimal_test_data"
+
+[trainer.validation_data]
+@baseline_data = "minimal_test_data"
+
+[trainer.logger]
+@baseline_loggers = "terminal_logger"
```

Comment:
Would love to interpolate this, but got a recursion error if I did `${logger}`. Any suggestions @KennethEnevoldsen?

Reply:
Hmm odd. Is it due to imports on parsing the config? (I don't see why parsing the config should lead to a recursion error).

I assume you did it like this:
```
[trainer]
 @baseline_trainers = "split_trainer"
 training_outcome_col_name = "outcome"
 validation_outcome_col_name = "outcome"
 logger = ${logger}
 ```

Reply:
Yep, exactly. Huh, we'll have to debug that, but I don't think it should block this merge :tu